reinforcement_learning/rl_deepracer_robomaker_coach_gazebo/deepracer_rl.ipynb

```diff
@@ -378,7 +378,6 @@
    "source": [
```

Comment:
<p>estimators_kwargs = {</p><p>&nbsp;&nbsp;"entry_point": "base_training_worker.py",</p><p>&nbsp;&nbsp;"source_dir": 'src',</p><p>&nbsp;&nbsp;"dependencies": ["common/"],</p><p>&nbsp;&nbsp;"role": sagemaker_role,</p><p>&nbsp;&nbsp;"train_instance_type": instance_type,</p><p>&nbsp;&nbsp;"train_instance_count": 1,</p><p>&nbsp;&nbsp;"output_path": s3_output_path,</p><p>&nbsp;&nbsp;"base_job_name": job_name_prefix,</p><p>&nbsp;&nbsp;"metric_definitions": metric_definitions,</p><p>&nbsp;&nbsp;"train_max_run": job_duration_in_seconds,</p><p>&nbsp;&nbsp;"hyperparameters": custom_hyperparameter,</p><p>&nbsp;&nbsp;"subnets": deepracer_subnets,</p><p>&nbsp;&nbsp;"security_group_ids": deepracer_security_groups,</p><p>}</p><p>looks like the arg names for sagemaker v1; we are prioritizing sagemaker v2. Is there any reason to use v1 convention?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
For backward compatibility. Some of the customers would have the old notebook running which uses sagemaker v1. Until the SageMaker team enforce everyone to move to the newer version we would like to support both.